### PR TITLE
SEO-173296 WPF Chart Annontations

### PR DESCRIPTION
**Pull request by Yvone-Atieno**
Hi @MallikaRK ,

Have renamed image name alt, and changed folder name.

Regards,
Yvone.

### CODE REVIEW

**Review by MallikaRK — APPROVED** [review]
Approved

**Review by Girija-Rajendran — APPROVED** [review]

File: wpf/Charts/Annotations.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ chart.Annotations.Add(annotation);
 
 {% endtabs %}
 
-![Annotation in WPF Chart](Annotation_images/wpf-chart-annotation.jpeg)
+![Annotation in WPF Chart](annotation_images/wpf-chart-annotation.jpeg)
```
Comment (Girija-Rajendran) [inline wpf/Charts/Annotations.md:49]: Changed in folder?
image

Comment (Yvone-Atieno) [reply wpf/Charts/Annotations.md:49]: Yes the folder is changed

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine.

**Pull request merged**